src/Juvix/Compiler/Concrete/Translation/FromSource.hs

```diff
@@ -82,8 +80,8 @@ fromSource e = mapError (JuvixError @ParserError) $ do
       Path Abs File ->
       Sem r (Module 'Parsed 'ModuleTop)
     goFile fileName = do
-      input <- getFileContents fileName
-      mp <- runModuleParser fileName input
+      input_ <- getFileContents fileName
+      mp <- runModuleParser fileName input_
```

Comment:
Unrelated (?) name changes.

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582